cmd/trillian_log_signer/main.go

```diff
@@ -47,13 +47,15 @@ import (
 	"github.com/google/trillian/util/election"
 	"github.com/google/trillian/util/election2"
 	etcdelect "github.com/google/trillian/util/election2/etcd"
+	"github.com/google/trillian/util/election2/sql"
```

Comment:
This comment relates to the package of the new code. I recommend `mysql` over `sql` to be clear at which implementation it is targeting, and for consistency with the flag names used in this file to pick it.